design/src/high-level/analysis-graphics.md

```diff
@@ -0,0 +1,101 @@
+# High-Level Analysis: Graphics
```

Comment:
I am liking this in general, but I do wonder if there can be more documentation around *why* a particular example is relevant graphically for Emergence,

It would also be helpful to have documentation that talks about why particular graphic design choices from an example are relevant for Emergence,

and in general, when discussing relevant graphic design choices, it would be nice to have links to examples, that way this document is self-contained as much as possible (we do have to manage link rot though...)

(Sorry, I hope I am not being too annoying here?)

@alice-i-cecile does the above make sense to you too? maybe it's fine as is....

Reply:
I agree, context and goals are really helpful and would be useful for each of these docs.